test/Interop/Cxx/stdlib/include-glibc.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-run-simple-swift(-Xfrontend -enable-cxx-interop)
```

Comment:
In the filename: s/stdlib/libc/ ?

(all filenames)

Reply:
Done.